tests/components/weheat/conftest.py

```diff
@@ -130,6 +130,8 @@ def mock_weheat_heat_pump_instance() -> MagicMock:
     mock_heat_pump_instance.indoor_unit_gas_boiler_state = False
     mock_heat_pump_instance.indoor_unit_electric_heater_state = True
 
+    mock_heat_pump_instance.async_get_status = AsyncMock()
```

Comment:
I believe if you make `mock_heat_pump_instance` an `AsyncMock(spec=...)`I think you don't even need this